backend/src/main.py

```
@@ -81,12 +89,24 @@ def internal_error(error):
-    # For eventlet, tasks are usually spawned directly
-    socketio.start_background_task(mempool_monitor_service.start_monitoring)
-    socketio.run(app, host='0.0.0.0', port=5000)
+def start_async_loop():
+    """
+    Starts an asyncio event loop in a background thread for monitoring and auto-trading.
+    """
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
 
+    # Schedule background tasks
+    loop.create_task(mempool_monitor_service.start_monitoring())
```

Ruff (0.15.6)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)